docs/userguides/compile.md

```diff
@@ -85,7 +85,14 @@ compile:
 ## Settings
 
 Generally, configure compiler plugins using your `ape-config.yaml` file.
-For example, when using the `vyper` plugin, you can configure settings under the `vyper` key:
+One setting that applies to many compiler plugins is `compiler_cache_folder`, which holds dependency source files the compiler uses when compiling your contracts.
+By default, the folder is in your `contracts/.cache` folder but there are times you may want to move this to another location.
```

Comment:
If any caveats, gotchas, or  findings from ape-solidity work goes, we should maybe include here.. like 

"NOTE: If using ape-solidity, your hashes are different now or wha have yous

Reply:
Will keep that in mind.  I don't have anything definitive to add at the moment, though.